doc/reference/reports.rst

```diff
@@ -65,9 +68,9 @@ Example::
         id="account_invoices"
         model="account.invoice"
         string="Invoices"
-        report_type="qweb-pdf"
         name="account.report_invoice"
-        file="account.report_invoice"
+        report_type="qweb-pdf"
+        print_report_name="(object._get_report_filename())"
```

Comment:
Maybe an example using the `%` string substitution syntax would better since that would be a more typical value?

Reply:
There is one example below for attachment already ;).
This way I show two different examples (and I keep the logic copy/pasted from the existing `account.account_invoices` report definition.

Reply:
Alright. Out of curiosity: why are there parentheses around the `_get_report_filename()` call?

Reply:
IMO they aren't needed necessary (and have no impact), but I'll check :D